launch/TalonTest.launch

```diff
@@ -1,7 +1,7 @@
 <launch>
 	<!-- joy node -->
 	<node respawn="true" pkg="joy" type="joy_node" name="joy_node" >
-		<param name="dev" type="string" value="/dev/input/js0" />
+		<param name="dev" type="string" value="/dev/input/js2" />
```

Comment:
What's the reason to go from js0 to js2? Or is this just the case of a change in values similar to USB ports

Reply:
Yeah just what the particular system wanted